qiskit_ibm_runtime/utils/backend_decoder.py

```diff
@@ -51,6 +58,7 @@ def configuration_from_server_data(
         return None
     try:
         decode_backend_configuration(raw_config)
+        filter_raw_configuration(raw_config, use_fractional_gates=use_fractional_gates)
```

Comment:
If it's loaded lazily another option would be adding lazy filtering by consuming the `backend.options.use_fractional_gates` in [here](https://github.com/Qiskit/qiskit-ibm-runtime/blob/da0a2d763d5592e8e3e472ecfcc17d7915b065b7/qiskit_ibm_runtime/ibm_backend.py#L271).

Reply:
Right, I had meant to do that but I left out the check of `backend.options.use_fractional_gates` and also got the logic of the filtering backwards. I fixed it in 598249e41679c15029e0aa19071b94f04a96e61f. Also, I added some testing in 7cdad8a96141f898ec804b7774159d4afbc745cf that catches the error I made -- it will fail the tests if the properties do not get loaded into the target.

Reply:
Outdated -- the properties filtering should work now.